geoh5py/objects/drillhole.py

```diff
@@ -719,11 +747,13 @@ def sort_depths(self):
                         child.values = child.format_values(child.values)[sort_ind]
 
                 if self.vertices is not None:
-                    self.vertices = self.vertices[sort_ind, :]
+                    self._vertices = self.vertices[sort_ind, :]
+                    self.workspace.update_attribute(self, "vertices")
 
                 if self.cells is not None:
                     key_map = np.argsort(sort_ind)[self.cells.flatten()]
-                    self.cells = key_map.reshape((-1, 2)).astype("uint32")
+                    self._cells = key_map.reshape((-1, 2)).astype("uint32")
+                    self.workspace.update_attribute(self, "cells")
 
 
 def compute_deviation(surveys: np.ndarray) -> np.ndarray:

```

Comment:
why not create staticmethods?

Are those function used anywhere else?

If so, I t would be better to create a drillhole_utils file in share.

Reply:
Sure, beyond scope of this PR. We can tackle on GEOPY-1627